doc/api/animation_api.rst

```diff
@@ -103,6 +103,7 @@ artist at a global scope and let Python sort things out.  For example ::
    import numpy as np
    import matplotlib.pyplot as plt
    from matplotlib.animation import FuncAnimation
+   from functools import partial
```

Comment:
```suggestion

```

Reply:
Aren't we using partial here?

Reply:
Not that I can see...

Reply:
Sorry, I got confused with the example below